picard/mbjson.py

```diff
@@ -383,8 +384,13 @@ def recording_to_metadata(node, m, track=None):
             add_isrcs_to_metadata(value, m)
         elif key == 'video' and value:
             m['~video'] = '1'
+    config = get_config()
     if m['title']:
         m['~recordingtitle'] = m['title']
+    if m['~recordingoriginaldate'] and (
+        config.setting["originaldate_use_recording"] or not m['originaldate']):
```

Comment:
Unrelated to this PR but this use-case of `get_config()` makes me think we perhaps need a shortcut:

```python
def get_config_setting(name):
   config = get_config()
   return config.setting[name]
```

Not sure about that, because it may lead to write:

```python
   if get_config_setting('x'):
        ....
   if get_config_setting('y'):
        ....
   if get_config_setting('z'):
        ....
```

which is likely less efficient than:

```python
   config = get_config()
   if config.setting['x']:
        ....
   if config.setting['y']:
        ....
   if config.setting['z']:
        ....
```

Thoughts?

Reply:
We could have that in addition for the simple cases, but IMHO there is not much difference between these two:

```python
get_config_setting('x')
get_config().setting['x']
```

That would also be slightly more efficient in this case, but not as readable I think.

And yes, if you need to access it multiple times it's better to get the config object once and reuse it. Or even not get it at all. We should sometimes see if we can avoid accessing config directly from helper functions but instead use specific parameters. That could also be an option here, leaving it to the caller of function to extract the proper value from config.

So e.g. we could have in this case a:

```
def recording_to_metadata(node, m, track=None, set_originaldate=True):
    ...
```

Of course the risk is that we get too many parameters or we have to route them through too many levels.